portal-api/api-tests/common-tests/role-authorisation-tests.js

```diff
@@ -28,6 +28,25 @@ const withRoleAuthorisationTests = ({ allowedRoles, getUserWithRole, makeRequest
   });
 };
 
+const malformedPayloadTests = ({ allowedRoles, getUserWithRole, makeRequestAsUser, successStatusCode }) => {
+  const notAllowedRoles = allRoles.filter((role) => !allowedRoles.includes(role));
+
+  if (notAllowedRoles.length) {
+    it.each(notAllowedRoles)('returns a 401 response for requests from a user with role %s', async (role) => {
```

Comment:
```suggestion
    it.each(notAllowedRoles)('should return a 401 response for requests from a user with role %s', async (role) => {
```
Can we also use the status code in the it

Reply:
Added.